oslo/torch/nn/parallel/data_parallel/__init__.py

```diff
@@ -1,3 +1,6 @@
+from oslo.torch.nn.parallel.data_parallel.data_parallel import (
+    DistributedDataParallel,
+)
 from oslo.torch.nn.parallel.data_parallel.zero import ZeroRedundancyOptimizer
 
-__all__ = ["ZeroRedundancyOptimizer"]
+__ALL__ = ["DistributedDataParallel", "ZeroRedundancyOptimizer"]
```

Comment:
```suggestion
__all__ = ["DistributedDataParallel", "ZeroRedundancyOptimizer"]
```
I think it is desirable to use small letters.
Check this https://docs.python.org/3/tutorial/modules.html#importing-from-a-package.

Reply:
Followed other parallel code